backend/src/test/java/ai/verta/modeldb/TestSequenceSuite.java

```diff
@@ -28,10 +25,10 @@
   RepositoryTest.class,
   CommitTest.class,
   MetadataTest.class,
-  DiffTest.class,
-  BlobEquality.class,
-  BlobProtoEquality.class,
-  DiffAndMerge.class,
+  //   DiffTest.class,
```

Comment:
Do these tests not exist anymore? Why are they commented out?

Reply:
Test class committed by mistake so removed from this PR